### PR TITLE
Virtual people

### PR DESCRIPTION
**Pull request by MyNameIsRomayne**
dreamseeker_qZYjnfMdSO

People who spawn in with the pods alive speak out loud in VR world

### CODE REVIEW

File: code/datums/Vspace.dm [hunk 1]
```diff
@@ -212,6 +212,11 @@ datum/v_space
 			virtual_character.real_name = "Virtual [user.real_name]"
 		user.mind.virtual = virtual_character
 		user.mind.transfer_to(virtual_character)
+		// We do this here because transfer_to overwrites the say_tree
+		if (!virtual_character.isghost)
+			virtual_character.ensure_say_tree().AddOutput(SPEECH_OUTPUT_SPOKEN)
+			virtual_character.ensure_say_tree().RemoveOutput(SPEECH_OUTPUT_DEADCHAT)
+			virtual_character.default_speech_output_channel = SAY_CHANNEL_OUTLOUD
```
Comment (Mister-Moriarty) [inline code/datums/Vspace.dm]: Could this be done on the virtual human's `New()`, or does that break setup?

File: code/modules/speech/core.dm [hunk 2]
```diff
@@ -157,7 +157,7 @@ Cleanup:
 
 Things To Implement:
 - AI
-- Virtual humans
+- Afterlife bar radio (:d for deadchat)
```
Comment (Mister-Moriarty) [inline code/modules/speech/core.dm]: Given that the deadchat PR is up, removing this may be best.

**Review by Mister-Moriarty — CHANGES_REQUESTED** [review]

File: code/modules/speech/core.dm (hunk 2 above)
Comment (MyNameIsRomayne) [reply code/modules/speech/core.dm]: *nod

File: code/datums/Vspace.dm (hunk 1 above)
Comment (MyNameIsRomayne) [reply code/datums/Vspace.dm]: That was my first attempt -- however, this proc seems to overwrite the say tree with the tree of the mob joining Vspace. Either the channels would need to be added here as-is, or transferring into a client would need to not overwrite the mob's say tree input/outputs.

TL;DR breaks setup

Comment (Mister-Moriarty) [reply code/datums/Vspace.dm]: That's odd; I don't think transferring clients transfers mob say tree outputs? It only transfers OOC modules and similar.

Comment (MyNameIsRomayne) [reply code/datums/Vspace.dm]: I stepped through this proc, in `user.mind.transfer_to(virtual_character)` it was happening

**Review by Mister-Moriarty — APPROVED** [review]
